### PR TITLE
sidebar and what we offer pages changes

### PR DESCRIPTION
**Pull request by alykhan02**
Built out about-us page
made changes to sidebar

### CODE REVIEW

File: src/components/VBSideBar.tsx
Comment (alykhan02) [inline src/components/VBSideBar.tsx:1]: Lines getting removed is the budget icon which Jessica said we dont need

File: src/navigation/authRoutes.tsx
Comment (alykhan02) [inline src/navigation/authRoutes.tsx:1]: Jessica just wanted to change the order and remove Courses

File: src/views/anon/intro/AboutUs.tsx
Comment (alykhan02) [inline src/views/anon/intro/AboutUs.tsx:1]: style changes (moving elements on the page)

File: src/views/anon/intro/WhatWeOffer.tsx
Comment (alykhan02) [inline src/views/anon/intro/WhatWeOffer.tsx:1]: had to build this page from scratch but repurposed bunch of components from Home/About Us page

File: src/views/anon/intro/WhatWeOffer.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                    <Grid item lg={6} sx={{height: '100%', width: '100%', padding: '0 80px 20px 80px', display: 'flex !important', alignItems: 'start !important', justifyContent: 'start !important', flexDirection: 'column !important'}}>
+                        <Typography variant='overline' fontWeight='bold' color='#B54708'>
+                            Overview
+                        </Typography>
+                        <Typography sx={{display: {xs: 'inline', xl: 'none'}}} variant='h3' gutterBottom>
+                        Beyond a co-working space
+                        </Typography>
+                        <Typography sx={{display: {xs: 'none', xl: 'inline'}}} variant='h3' gutterBottom>
+                        Beyond a co-working space
+                        </Typography>
```
Comment (S-Makrod) [inline src/views/anon/intro/WhatWeOffer.tsx]: These have exact same content and styling, so I don't think we need to vary over display sizes

File: src/views/anon/intro/WhatWeOffer.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                <Box sx={{overflowX: 'hidden', marginTop: '20px', display: 'flex', alignItems: 'center', flexDirection: 'column', textAlign: 'center'}}>
+                    <Typography variant='overline' fontWeight='bold' color='#B54708'>
+                        Features
+                    </Typography>
+                    <Typography sx={{display: {xs: 'inline', xl: 'none'}}} variant='h3' gutterBottom>
+                        Supercharge your startup experience
+                    </Typography>
+                    <Typography sx={{display: {xs: 'none', xl: 'inline'}}} variant='h3' gutterBottom>
+                        Supercharge your startup experience
+                    </Typography>
```
Comment (S-Makrod) [inline src/views/anon/intro/WhatWeOffer.tsx:121]: These have exact same content and styling, so I don't think we need to vary over display sizes

**Review by S-Makrod — APPROVED** [review]
Just a few small changes, otherwise good to merge

**Comment by hmxxxx** [comment]
[[FE/BE] What we offer page](https://trello.com/c/PMKt7Tqz/34-fe-be-what-we-offer-page)

**Pull request merged**